CHANGELOG.md

```diff
@@ -1,9 +1,51 @@
 # Changelog
+## Friday 22nd July 2022
+### Normaliser
+We're working on our normaliser and we'll be rolling out some changes next week. This will allow the likes of `cm` to be pronounced correctly and also will work for dates and some times. 
+
+### Billing enhancements
+We've made some billing enhancements and improving to our billing systems.
+## Friday 15th July 2022
+
+### OpenAPI
+* We launched [OpenAPI](https://aflorithmic.github.io/openapi/) so you can see in [Swagger](https://swagger.io/) style our endpoints. If you're used to this workflow let us know we'd love to talk some more about it. Feel free to reach out to peadar[at]aflorithmic[dot]ai
+
+### SuperOrg
+A common customer request was - I want to understand what my customers usage is. For example if you're a distribution partner or agency you'll encounter this workflow. 
+
+This allows you to see usage of your child organisations :) Have a look below. It's in production so you can 
+![SuperOrg](images/superorg_with_childorgs_usage.png) 
+### SyncTTS
+* When you use syncTTS with `metadata=True` there will also be a url generated
+
+```python 
+response = apiaudio.SyncTTS.create(text="Hello", voice="eth-einstein-v2", metadata=True)
```

Comment:
May we shouldn't use private customer voices in the changelog :D

Reply:
Ohh yeah what works as a voice?